src/test/mir-opt/instrument_coverage.bar.InstrumentCoverage.diff

```diff
@@ -72,6 +14,10 @@
                                            // + span: /the/src/instrument_coverage.rs:20:5: 20:9
                                            // + literal: Const { ty: bool, val: Value(Scalar(0x01)) }
           return;                          // scope 0 at /the/src/instrument_coverage.rs:21:2: 21:2
++     }
++ 
++     bb1 (cleanup): {
```

Comment:
I think this might have been added by the patch implementation. Since I'm only inserting a statement, maybe I don't need to use patch at all.

Reply:
Fixed